firmware/shared/controls/tvFactor_test.cc

```diff
@@ -0,0 +1,44 @@
+#include <iostream> 
+
+#include "tvFactor.h"
+
+#include "testing.h"
+
+int main() {
+    using namespace ctrl;
+
+    ASSERT_CLOSE(
+        EnableTorqueVectoring(26.3, 0.2),
+        26.3);  // Should return 30.0 because DI_b_EnableTorqueVectoring > 0
+    ASSERT_CLOSE(
+        EnableTorqueVectoring(21.3, 0.0),
+        0.0);  // Should return 0.0 because DI_b_EnableTorqueVectoring <= 0
+    ASSERT_CLOSE(
+        EnableTorqueVectoring(-20.0, 1.5),
+        -20.0);  // Should return -20.0 because DI_b_EnableTorqueVectoring > 0
+    ASSERT_CLOSE(
+        EnableTorqueVectoring(12.0, -6.2),
+        0.0);  // Should return 0.0 because DI_b_EnableTorqueVectoring <= 0
+
+    ASSERT_CLOSE(CreateTorqueVectoringFactor(5.0), 0.934);
+    ASSERT_CLOSE(CreateTorqueVectoringFactor(10.0), 0.87);
+    ASSERT_CLOSE(CreateTorqueVectoringFactor(13.2), 0.83032);
+    ASSERT_CLOSE(CreateTorqueVectoringFactor(23.3), 0.70476);
+    ASSERT_CLOSE(CreateTorqueVectoringFactor(27.3), 0.683);
+
+    auto [leftFactor1, rightFactor1] = AdjustTorqueVectoring(15.2, 3.6);
```

Comment:
use scoping notation since you're creating variables. I know it's not a problem for this set of tests, but it's good practice to "delete" the variables after they go out of scope. 

To do this, wrap lines 29-31 in `{ }` like this

```c++
{
    auto [leftFactor1, rightFactor1] = AdjustTorqueVectoring(15.2, 3.6);
    ASSERT_CLOSE(leftFactor1, 1.0); 
    ASSERT_CLOSE(rightFactor1, 3.6);
}
```

Then the left/right factor variables are not accessible after the block of code, so you don't even need to number them. Also fix the variable naming as per the guidelines